Pod/Classes/WPMediaCollectionViewController.m

```diff
@@ -351,6 +329,54 @@ - (UICollectionViewCell *)collectionView:(UICollectionView *)collectionView cell
     return cell;
 }
 
+- (CGSize)collectionView:(UICollectionView *)collectionView
+                  layout:(UICollectionViewLayout *)collectionViewLayout
+referenceSizeForHeaderInSection:(NSInteger)section
+{
+    if ( [self isShowingCaptureCell] && self.showMostRecentFirst)
+    {
+        return CameraPreviewSize;
+    }
+    return CGSizeZero;
+}
+
+- (CGSize)collectionView:(UICollectionView *)collectionView
+                  layout:(UICollectionViewLayout *)collectionViewLayout
+referenceSizeForFooterInSection:(NSInteger)section
+{
+    if ( [self isShowingCaptureCell] && !self.showMostRecentFirst)
+    {
+        return CameraPreviewSize;
+    }
+    return CGSizeZero;
+}
+
+- (UICollectionReusableView *)collectionView:(UICollectionView *)collectionView
+           viewForSupplementaryElementOfKind:(NSString *)kind
+                                 atIndexPath:(NSIndexPath *)indexPath
+{
+    if ((kind == UICollectionElementKindSectionHeader && self.showMostRecentFirst) ||
```

Comment:
I tested this and it looks that it's only called for non zero sized header and footers, but I will prefer to keep the if to better future proof the code in case of new layouts or adding new headers/footers